src/Components/v2/Lightbox/Lightbox.tsx

```diff
@@ -281,15 +288,26 @@ export class Lightbox extends React.Component<LightboxProps, LightboxState> {
     const modifiedChildren = React.Children.map(
       children,
       (child: React.ReactElement<any>) => {
-        return React.cloneElement(child, {
-          ...(enabled && {
-            style: { cursor: "zoom-in" },
-            onClick: this.show.bind(this),
-            // Used by view-in-room
-            "data-type": "artwork-image",
-            "data-is-default": isDefault,
-          }),
-        })
+        const {
+          props: { src },
+        } = child
+        return (
+          enabled && (
+            <Flex
+              justifyContent="center"
+              alignItems="center"
+              height="60vh"
```

Comment:
If the height of the parent flex container is specified here, do we also need to specify `60vh` in `StyledImage`? Or could it be `100%`?

Reply:
`100%` would be better, just updated it!